src/components/MainMenu/MainMenu.tsx

```diff
@@ -96,60 +96,62 @@ const MainMenu: React.FC = () => {
-                          suffixClass={'__rightdown'}
-                          head={
-                            <li className="main-menu__icon main-menu__user--active">
-                              <ReactSVG path={userImg} />
-                            </li>
-                          }
-                          content={
-                            <ul className="main-menu__dropdown">
-                              <li data-cy="mobileMenuMyAccountLink">
-                                <Link to={appPaths.accountUrl}>My Account</Link>
-                              </li>
-                              <li data-cy="mobileMenuOrderHistoryLink">
-                                <Link to={appPaths.orderHistoryUrl}>
-                                  Order history
-                                </Link>
-                              </li>
-                              <li data-cy="mobileMenuAddressBookLink">
-                                <Link to={appPaths.addressBookUrl}>
-                                  Address book
-                                </Link>
-                              </li>
+                      <Media
+                        query={{ maxWidth: smallScreen }}
+                        render={() => (
+                          <>
+                            {user ? (
+                              <MenuDropdown
+                                suffixClass={"__rightdown"}
+                                head={
+                                  <li className="main-menu__icon main-menu__user--active">
+                                    <ReactSVG path={userImg} />
+                                  </li>
+                                }
+                                content={
+                                  <ul className="main-menu__dropdown">
+                                    <li data-cy="mobileMenuMyAccountLink">
+                                      <Link to={appPaths.accountUrl}>
+                                        My Account
+                                      </Link>
+                                    </li>
+                                    <li data-cy="mobileMenuOrderHistoryLink">
+                                      <Link to={appPaths.orderHistoryUrl}>
+                                        Order history
+                                      </Link>
+                                    </li>
+                                    <li data-cy="mobileMenuAddressBookLink">
+                                      <Link to={appPaths.addressBookUrl}>
+                                        Address book
+                                      </Link>
+                                    </li>
+                                    <li
+                                      onClick={handleSignOut}
+                                      data-cy="mobileMenuLogoutLink"
+                                    >
+                                      <ReactSVG path={userImg} />
```

Comment:
I guess it shouldn't be changed from `Log Out` string, isn't it?

Reply:
It wasn't, rebase conflicts.